packages/forms-web-app/src/dynamic-forms/s78-questionnaire/journey.js

```diff
@@ -30,154 +33,131 @@ class S78Journey extends Journey {
 			journeyTitle
 		);
 
+		const meetsCondition = meetsConditionBuilder(response, questions);
```

Comment:
This may be slightly pedantic but I wonder whether the naming should be changed from `meetsCondition` to `questionHasAnswer` as this doesn't evaluate an arbitrary condition. As far as I can see, it always checks whether a given question has a given answer. That would mean that, when it's used, it would look something like this (also incorporating Ben's change):
`.withCondition(questionHasAnswer(questions.changesListedBuilding, 'yes'))`
To me, that makes it slightly more obvious what it's doing.

Reply:
sounds good - have renamed the functions